CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [8.0.0] - 2018-01-19
+### Added
+- Added `scriptworker.cot.verify.get_jsone_template`, because action tasks use actions.json instead of .taskcluster.yml
```

Comment:
Okay, so that was why you had multiple iterations? afaik, the decision task copies `.taskcluster.yml` into the `actions.json` that it generates...

Reply:
Correct. Multiple iterations works! But this is what the original process does, so it's more likely to stay a faithful reproduction.